apps/docs/content/components/OrderedList.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Ordered List
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/branch/NzBMPIfPrFU9I87d3k7KVq/Primer-Brand?node-id=4045%3A17560&t=W8B9ArDngbzJCC2m-0'
+source: https://github.com/primer/brand/tree/main/src/List/OrderedList/OrderedList.tsx
+storybook: '/brand/storybook/?path=/story/components-orderedlist--default'
+description: Use the OrderedList component to render a list of ordered items.
+---
+
+```js
+import {OrderedList} from '@primer/react-brand'
+```
+
+## Examples
+
+### OrderedList
```

Comment:
For consistency with other components docs

```suggestion
### Default
```

Reply:
Updated in next push